plugins/katello/developers.md

```diff
@@ -127,105 +127,40 @@ Prior to branching the Katello repositories, the release nanny should figure out
-
-```
-cd katello
-git fetch origin
-git checkout master
-git reset origin/master --hard
-tito tag
-cd deploy/
-tito tag
-git checkout -b KATELLO-X.Y
+./tools.rb setup-environment --github-username <username> configs/katello_21.yaml
 ```
 
-Update the releasers and tito configuration for each package (some repositories contain multiple packages, see Step 3 for list):
+You will find the repositories checked out to the versions specified in the release config in `repos/katello_X.Y.0/`.
 
-```
-VER=X.Y sed -i "s#/katello-nightly-rhel/#/katello-$VER-rhel/#g" rel-eng/*
-git commit -a -m 'Updating rel-eng for X.Y'
-git push origin KATELLO-X.Y
-```
+### Step 5: Get Your Branch On
 
-Now we set master to start from the version after the one we are working on releasing X.Y+1, for each package, ensuring you are in the directory the spec file exists:
+On the date you previously warned the community branching would occur, ensure any project dependencies are updated, including gem version constraints, and branch the repositories:
 
 ```
-git checkout master
+./tools branch configs/katello_XY.yaml
 ```
 
-For any of the gems, update the gem version file and the spec file to X.Y.0 and make sure to reset the release to 1. For example,
+This will also branch the documentation at [theforeman.org](https://theforeman.org/plugins/katello).
 
-```
-vim lib/katello/version.rb
-vim rubygem-katello.spec
-git commit -a -m 'Version bump to X.Y.0'
-```
-
-Now for all of the repositories including the gems, tag and push the starting place for the next version:
-
-```
-tito tag --use-version=X.Y+1.0
-git push origin master && git push --tags
-```
-
-For `hammer-cli-katello`:
-
-1. If there are no changes since last version do nothing
-2. If there are changes, bump the version, release the gem, tag and build in koji
-
-Lastly, branch documentation
-
-Now we are ready to branch the documentation. This is done separately from the other branching due to the fact that we do not tag or generate an RPM of the documentation website.
-
-```
-git clone https://github.com/Katello/katello.org
-cd katello.org
-git checkout -b KATELLO-X.Y
-git push origin KATELLO-X.Y
-```
-
-Check that the CI job that builds the site (http://ci.theforeman.org/job/deploy_katello_site/) has been kicked off or kick it off manually if needed. Once it passes, visit [http://www.katello.org](http://www.katello.org) and check that X.Y is available from the drop down list. If it did not build correctly, you will need to checkout the `deploy` branch and test it locally.
+Check that the [CI job that builds the site](http://ci.theforeman.org/job/deploy_katello_site/) has been kicked off or kick it off manually if needed. Once it passes, visit [the Katello homepage](http://www.theforeman.org/plugins/katello) and check that X.Y is available from the drop down list. If it did not build correctly, you will need to checkout the `deploy` branch and test it locally.
```

Comment:
This is no longer true, and can be removed entirely given this is all inside theforeman.org repository.